src/components/BinaryButton.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="binary-button">
+    <div @click="onClick(true)" :class="{'binary-button-selected': leftChosen}">
+      <img v-if="leftChosen" class="confirmation-icon" src="../assets/images/check.svg" />
+      <div>
+        {{ choices[0] }}
+      </div>
+      <img v-if="leftChosen" class="confirmation-icon hidden" src="../assets/images/check.svg" />
```

Comment:
this is here to offset the checkmark and so everything stays centered. https://stackoverflow.com/questions/38948102/center-one-and-right-left-align-other-flexbox-element